dev/kustomize.py

```
@@ -0,0 +1,73 @@
+            dst=tmp_dir,
+            dirs_exist_ok=True,
+        )
+        
+        # If a image has been provided find the digest and run run the 'kustomize' tool to set it, this way the image
+        # used will always be the latest that was built and pushed, regardless of the image pull policy.
+        if image is not None:
+            code, image = command.eval(
+                args=[
+                    "podman",
+                    "inspect",
+                    f"--format={{{{ index .RepoDigests 0 }}}}",
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)